### DISCUSSION TITLE
How to use react-schema-editor? Example of using can be very helpfull

### DISCUSSION THREAD

**Discussion by dmushkov** [alibaba/formily#1157] [Q&A 2.x] [answered]
ERROR in ../node_modules/@formily/react-schema-editor/src/index.tsx 17:25
Module parse failed: Unexpected token (17:25)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| import './index.css'
| 
> export const SchemaEditor: React.FC<{
|   className?: string
|   schema: any
 @ ./index.tsx 24:0-64 181:42-54

**Comment by jsdocker** [comment]
I have the same question

**Comment by janryWang** [comment, marked as answer]
这个包已经废弃了，请使用https://github.com/alibaba/formily-editor
